docs/fast_data/configuration/config_maps/erSchema.md

```diff
@@ -19,11 +19,31 @@ The ER Schema or Entity-Relation Schema defines the relationships between the co
 Don't forget to [declare your relationships both ways](#direction-of-the-relationships)! 
 :::
 
-## Syntax
 
-The ER Schema is made of the following fields:
+## Configuration Properties
 
-```json title="erSchema.json"
+The configuration file has two required fields:
+- _version_, the current configuration version, which determines the syntax and semantics of the rest of the configuration. The following properties follow the `1.0.0` syntax version.
+- _config_, the whole ER Schema config, detailed above.
+
+| Name | Type | Required | Default | Description |
+|------|------|----------|---------|-------------|
+| COLLECTION | String | - | - | Name of a collection of the System of Records. There should be a `COLLECTION` object for each collection of the System of Records. |
+| outgoing | String[] | - | - | List of all the related collections. |
+| RELATED_COLLECTION | String | - | - | Name of a collection related to the `COLLECTION`. There should be one `RELATED_COLLECTION` object for each collection related to `COLLECTION`. |
+| conditions | Condition[] | - | - | List of conditions that connect the `COLLECTION` and the `RELATED_CONNECTION`. |
+| CONDITION_NAME | String | - | - | Name of the condition, this is purely for debug purposes and we suggest using the following naming convention: `COLLECTION_to_RELATED_COLLECTION` (eg. dish_to_order_dish) |
+| oneToMany | boolean | - | false | Specifies if there will be **more than one** `RELATED_COLLECTION` document related to a `COLLECTION` document. If false we will assume there will be **only one** `RELATED_COLLECTION` document related to another `COLLECTION` document. |
+| RELATED_COLLECTION | String | - | - | Name of a collection related to the `COLLECTION`. There should be one `RELATED_COLLECTION` object for each collection related to `COLLECTION`. |
+| condition | Condition | - | - | Object literal containing the condition. |
+| RELATED_COLLECTION_FIELD_NAME | String | - | - | A field name of the `RELATED_COLLECTION` |
+| COLLECTION_FIELD_NAME | String | - | - | A field name of the `COLLECTION` |
```

Comment:
Not sure about this, let me know if it's ok for you

Reply:
@danibix95 could you pls have a look?

Reply:
I'm not sure a table is the best format to show/explain the structure of a JSON

Reply:
Okok, I made the same also in projection_changes.md and in kakfa_projection_updates.md because I thought that could be unified with other configurations inside the various sections (even if maybe in other cases they were not a json), but I'm switching back

Reply:
Done